ml-agents/mlagents/trainers/learn.py

```diff
@@ -101,6 +101,14 @@ def _create_parser():
         type=int,
         help="Number of parallel environments to use for training",
     )
+
+    argparser.add_argument(
+        "--ghost-swap",
```

Comment:
This needs a less confusing name and a less confusing description. (If such a thing exists)

Reply:
How is "team-change"? Check it out now @vincentpierre